core/src/main/scala/cromwell/util/JsonEditor.scala

```diff
@@ -87,11 +87,22 @@ object JsonEditor {
   def logs(json: Json): ErrorOr[Json] = {
     val inputsAndOutputs = NonEmptyList.of("outputs", "inputs")
     val shardAttemptAndLogsFields = NonEmptyList.of("shardIndex", "attempt", "stdout", "stderr", "backendLogs")
+
+    def removeSubworkflowCalls(calls: JsonObject): JsonObject = calls.filter {
+      case (_, json) => ! json.asArray.get.head.asObject.exists(_.contains(subWorkflowIdKey))
```

Comment:
`json.asArray.get.head` why do we take only `head` here?

Reply:
My assumption was that all calls within this array would have the same "shape": either they would all be subworkflows or regular jobs. Even if that assumption is correct I should probably add a comment to that effect. 🙂

Reply:
and actually I may pull this out to a function as I'm finding myself needing it again in BA-6087 work. 🙂